types/index.d.ts

```diff
@@ -1,8 +1,8 @@
 // TypeScript Version: 3.7
 
-import {Element, Node, Root} from 'xast'
+import * as xast from 'xast'
 
-type Children = string | Node | number | Children[]
+type Children = string | xast.Node | number | Children[]
```

Comment:
I’m confused about this one. Can any `xast.Node` be a child? I.e. the following is valid:

```ts
h('foo', null, { type: 'whatever' })
```

TBH I don’t even understand why this is exported in `@types/xast`. It’s a direct export of `unist.Node`.

Reply:
It is valid https://runkit.com/christianmurphy/5fbc319108e0d1001a633d4d

---

> I don’t even understand why this is exported in

"this" as in `Children`?
I see it as a utility type, which is more complex than `Node`.
"this" as in `xast.Node` itself?
It's a convenience type, it doesn't need to be re-exported, but I don't see harm in re-exporting it.
Do you have a particular concern?

Reply:
I meant _“this”_ as in `xast.Node` itself.

I moved the issue for fixing `Children` to #8.

I moved the discussion of the removal of `xast.Node` to https://github.com/syntax-tree/unist/discussions/30